ui/src/app/pages/organization/organization.service.ts

```diff
@@ -32,23 +32,22 @@ export class OrganizationService {
 
```

Comment:
is this service is used anywhere? should be removed

Reply:
there will be a separate task for organization module deleting